docs/pages/upgrading/agent-managed-updates/bootstrap-ec2-agent.mdx

```diff
@@ -0,0 +1,333 @@
+
+It assumes you'll join the agent to your cluster during first boot (token or delegated join method)
+and that your cluster has Managed Updates v2 enabled.
+
+Delegated joins avoid shipping any secret token in user data. You create a named token resource that
+encodes context rules (your AWS account, role ARNs, regions, etc.), then the agent proves its identity
+to the Auth Service using cloud-issued credentials. Your agent can fetch IAM credentials
+(e.g., EC2 instance profile, IRSA, or env vars).
+
+In this guide, we use the `teleport-update` binary for installation.
+The `teleport-update enable` command installs Teleport at the version advertised by the cluster and enables
+Managed Updates v2 on the host. It also creates the necessary systemd units for Teleport, along with a
+`teleport-update` timer that periodically runs `teleport-update update`. In addition, it saves most of the
+flags you pass in (such as -g, -p, or -b), so that running the command again will update the stored settings
+rather than requiring you to re-enter them.
+
+For full Managed Updates v2 instructions, see [Managed Updates forAgents (v2)](./agent-managed-updates.mdx).
+
+Every agent belongs to an update group (e.g., development, staging, prod), unless the `default` group is used.
+The cluster-side `autoupdate_config` resource defines when each group may update.
+Cloud or self-hosted clusters use the same schedule model. Self-hosted clusters also set the desired
+version via `autoupdate_version` resource.
+
+## Define groups and schedules
+
+Pick simple, meaningful groups like `development`, `staging`, `production`. Then model your rollout windows
+and sequencing in `autoupdate_config`:
+
+```yaml
+# autoupdate_config.yaml
+kind: autoupdate_config
+metadata:
+  name: autoupdate-config
+spec:
+  agents:
+    mode: enabled
+    strategy: halt-on-error
+    schedules:
+      regular:
+        - name: <Var name="development" />
+          days: ["Mon","Tue","Wed","Thu"]
+          start_hour: 4   # UTC
+        - name: staging
+          days: ["Mon","Tue","Wed","Thu"]
+          start_hour: 5
+          wait_hours: 24  # run a day later than development
+```
+
+This resource is used to controls the update schedule by defining different groups, sequence and upgrade window.
+
+In the example, we define two update groups: `development` and `staging`. Upgrades are allowed on Monday, Tuesday,
+Wednesday, and Thursday.
+
+The first group, `development`, must start the upgrade process at 04:00 UTC. Once it is completed and all
+agents in the `development` group are upgraded, the next group, `staging`, begins the upgrade process with
+a 24-hour delay at 05:00 UTC.
+
+If any agent in the `development` group fails, the upgrade process must stop. This behavior is controlled by
+the `halt-on-error` strategy value.
+
+Run the following command to create or update the resource:
+
+```code
+$ tctl create -f autoupdate_config.yaml
+```
```

Comment:
I don't think this section is required for users to follow the guide, we can do everything with the `default` group name. Maybe we can move this to the end of the guide in the "Next steps" section.

Reply:
Also we should add a note about the fact that config changes take into effect for the next version change, so users are not confused when they see their changes having no effect immediately.